css/documentation.css

```diff
@@ -139,6 +139,11 @@ a {
   flex-wrap: wrap;
 }
 
+#navigation .component-examples {
```

Comment:
If you can use only class that would be better, using id increases its specificity, would be tough to override it in the future if you are using Component Library in other projects.

Reply:
Hey Naman, thanks for pointing this out. I have fixed it and committed the change. :)